core/ir/aarch64/instr_create_api.h

```diff
@@ -463,12 +463,12 @@
  * they just need to know whether they need to preserve the app's flags, so maybe
```

Comment:
Please enable the count-ctis tests which use this, which will add regression tests.

Reply:
Oh, didn't realize it needed more: was just grepping for tests that use cbr.  Makes sense to separate out the mbr.  Is it easy to separate in the test?  Or just separately locally in the test and confirm cbr works and state that in the PR description and say that the test will be enabled soon when mbr is added and then enable the test in a separate PR for mbr, so long as that comes in relatively soon (i.e., not months later with no cbr test in the meantime).

mbr is supposed to obtain a real address so that sounds like something is wrong there.